addons/mail/static/src/js/views/activity/activity_renderer.js

```diff
@@ -197,12 +199,14 @@ class ActivityRenderer extends AbstractRendererOwl {
     }
 }
 
-ActivityRenderer.components = {
-    ActivityRecordAdapter,
-    ActivityCellAdapter,
-    KanbanColumnProgressBarAdapter,
-};
-ActivityRenderer.template = 'mail.ActivityRenderer';
+Object.assign(ActivityRenderer, {
+    components: {
+        ActivityRecordAdapter,
+        ActivityCellAdapter,
+        KanbanColumnProgressBarAdapter,
+    },
+    template: 'mail.ActivityRenderer',
+});
```

Comment:
changes in this file are out-of-scope of discuss, but I don't think there are issues with this change